scss/detail.scss

```diff
@@ -38,18 +38,24 @@ $lower-screen-width: calc(#{$book-badge-width} * 0.7);
 .book-cover-page {
   background-color: $white;
   box-shadow: 4px 4px 10px 0 $black-shadow;
-  display: flex;
-  height: 416px;
+  display: grid;
+  grid-template-areas: 
+    "area1 area1"
+    "area2 area2"
+    "area3 area4";
```

Comment:
Please add semantic names to areas, like `book-image`, `title`, etc. try use same or similar class name.